app/controllers/sites_controller.rb

```diff
@@ -1,5 +1,17 @@
 class SitesController < ApplicationController
   before_action :set_site
-  load_and_authorize_resource
+  load_and_authorize_resource instance_variable: :site, class: 'Site' # descendents still auth Site
   layout 'dashboard'
+
+  def update
```

Comment:
It strikes me as a little strange that this base class exposes an `#update` method that will get inherited by all the subclasses. Maybe we should have a separate site banner controller?